lib/l10n/app_cs.arb

```diff
@@ -104,5 +104,6 @@
     "passwordResetExplanation": "Na zadanou emailovou adresu vám dojde odkaz pro obnovení hesla.",
     "passwordResetConfirmation": "Email pro obnovu hesla byl úspěšně odeslán.",
     "invalidEmailAddressError": "Emailová adresa je ve špatném formátu.",
-    "userNotFoundError": "Uživatel s touto emailovou adresou neexistuje."
+    "userNotFoundError": "Uživatel s touto emailovou adresou neexistuje.",
+    "version": "Verze",
```

Comment:
Bohuzel tady asi nejde pouzit trailing commu
Nadava mi to ni kdyz pustim `flutter pub get`:

```
FormatException: Unexpected character (at line 109, character 1)
```